codegen/jennies/dstutil.go

```diff
@@ -10,6 +10,23 @@ import (
 	"github.com/dave/dst/dstutil"
 )
 
+func addGenComments() dstutil.ApplyFunc {
+	return func(c *dstutil.Cursor) bool {
+		switch c.Node().(type) {
+		case *dst.TypeSpec:
+			c.Parent().Decorations().Start.Append("// +k8s:openapi-gen=true")
+		case *dst.StructType:
+			// TODO: is there a way of getting to the typedef from the struct cursor?
+			/* c.Parent()
+			//c.Parent().(*dst.TypeSpec).Decs.Start = []string{"// +k8s:openapi-gen=true"}
+			fmt.Println(c.Parent().Decorations())
+			fmt.Printf("%#v\n", c.Parent())
+			x.Decs.Start = append(x.Decs.Start, "// +k8s:openapi-gen=true") */
```

Comment:
Just for my understanding, is this going to be a blocker or a nice-to-have? IIUC `TypeSpec` should cover all types – did I get it wrong?

Reply:
Nice-to-have--right now it ends up putting the k8s codegen annotation on every type, even when not applicable. Doesn't hut anything, but makes the code messier.

Reply:
Got it. Do you mind leaving mentioning explicitly in a comment?